fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/MatchExprOperator.java

```diff
@@ -24,20 +25,33 @@
 import java.util.Objects;
 
 public class MatchExprOperator extends ArgsScalarOperator {
-    public MatchExprOperator(ScalarOperator... arguments) {
+    private List<ScalarOperator> arguments;
```

Comment:
duplicate field

Reply:
Thanks for your comment. I've removed duplicated field now. @Seaven